packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/quanticGeneratedAnswer.css

```diff
@@ -4,7 +4,7 @@
 }
 
 .generated-answer__badge {
-  background-color: var(--lwc-brand-background-primary, #eef4ff);
-  color: var(--lwc-brand-primary, #1b96ff);
+  background-color: var(--lwc-colorBackgroundButtonDefaultHover, #eef4ff);
+  color: var(--lwc-brandPrimary, #1b96ff);
```

Comment:
did the name change? from `--lwc-brand-primary` to `--lwc-brandPrimary` or was it a mistake?

Reply:
it was a mistake.